lib/instances/BaseInstance.lua

```diff
@@ -204,6 +212,16 @@ function BaseInstance.prototype:WaitForChild(name, delay)
 	return self:FindFirstChild(name)
 end
 
+function BaseInstance.prototype:_PropagateAncestryChanged(child, parent)
+	self.AncestryChanged:Fire(child, parent)
+
+	local children = getmetatable(self).instance.children
+
+	for c in pairs(children) do
```

Comment:
Can we spell this out as `child` and potentially rename this function's `child` parameter?

Reply:
helll yea dude i got u